src/overview/reducer.js

```diff
@@ -151,6 +154,16 @@ const changeHasBookmark = (state, index) => {
     return { ...state, searchResult }
 }
 
+const setShowTooltip = (state, isShowTooltip) => {
+    return {
+        ...state,
+        showTooltip:
+            isShowTooltip === undefined
```

Comment:
no need to complicate the reducer like this. Just simply set the new state to be whatever the `isShowTooltip` payload is (should always be `true` or `false`; can't find anywhere in the code which dispatches it with no payload)